docker/compose-local-dev-traces-v2.overrides.yml

```diff
@@ -68,7 +68,7 @@ services:
   prover-v3: # prover compatible with the traces from zkbesu
     container_name: prover-v3
     hostname: prover-v3
-    image: consensys/linea-prover:${PROVER_TAG:-bba9677}
+    image: consensys/linea-prover:${PROVER_TAG:-8544ea1}
```

Comment:
update the image tag of prover-v3 to cope with the recent change of prover config